packages/fluentui/styles/src/mergeThemes.ts

```diff
@@ -108,10 +105,7 @@ export const mergeComponentStyles__PROD = (
   return result;
 };
 
-export const mergeComponentStyles__DEV = (
-  stylesA: ComponentSlotStylesInput | null | undefined,
-  stylesB: ComponentSlotStylesInput | null | undefined,
-): ComponentSlotStylesPrepared => {
+export const mergeComponentStyles__DEV: typeof mergeComponentStyles = (stylesA, stylesB) => {
```

Comment:
This PR discovered an issue that `__PROD` & `__DEV` functions had different interfaces 🙄